utils/bootstrap.py

```diff
@@ -65,11 +65,11 @@
         'url': 'https://prusa-buddy-firmware-dependencies.s3.eu-central-1.amazonaws.com/bootloader-mini-v2.0.2-FF7C65BF-1853-43B3-869C-C846FA39AB75.zip',
     },
     'mini404': {
-        'version': '0.9.3',
+        'version': 'e72651a',
```

Comment:
určitě to má nějaký důvod, jen škoda že to není tak lehce člověkem čitelný jako "0.9.3"

Reply:
I can always cut a new release to save everyone's sanity :)

Reply:
@davidnovacek yes, as @vintagepc mentions, it is because the commit wasn't properly released :)